src/store/rootReducer.ts

```diff
@@ -1,9 +1,10 @@
-import {Action, combineReducers} from '@reduxjs/toolkit';
+import {combineReducers} from '@reduxjs/toolkit';
+import {ticketReducer, TicketAction} from '@modules/ticket/reducer';
 
 export const rootReducer = combineReducers({
```

Comment:
Since we have just one reducer, I see no need for combineReducers, But for code structure and scaling this is fine